lib/moonshot/default_strategy.rb

```diff
@@ -1,13 +1,12 @@
 module Moonshot
   module ParameterStrategy
-    # Default strategy: grabbing every parameter
-    # from the parameter file.
+    # Default strategy: use parameter defined in the parameter file
     class DefaultStrategy
-      def parameters(_, overrides)
-        overrides.map do |key, _|
+      def parameters(params, _, _)
+        params.map do |key, _|
           {
             parameter_key: key,
-            parameter_value: overrides[key],
+            parameter_value: params[key],
```

Comment:
Any reason for doing this over having the map block yield value and putting value here? Pretty sure params[key] is just going to end up doing another hash look up for no reason.